api/src/main/java/net/jqwik/api/Arbitraries.java

```diff
@@ -454,7 +454,7 @@ public static CharacterArbitrary chars() {
 	 * @return a new arbitrary instance
 	 */
 	@API(status = MAINTAINED, since = "1.1.1")
-	public static <T> Arbitrary<T> create(Supplier<T> supplier) {
```

Comment:
I'm not sure that methods that previously returned <T> should now return <T extends @Nullable...>. It might be the right choice, but then it should be a different PR IMO.

Reply:
Do you mean you want forbid creating `Arbitrary<String?>` with this method?

Consider the use case:  `Arbitraries.create(() -> random.nextBoolean() ? "hi" : null)`.
Previous declaration (`<T>`) forbids such use cases, however, they are pretty valid.